modules/gapi/samples/one_vpl_source_to_bgr_conv.cpp

```diff
@@ -0,0 +1,66 @@
+#include <algorithm>
+#include <fstream>
+#include <iostream>
+#include <cctype>
+#include <tuple>
+#include <memory>
+
+#include <opencv2/imgproc.hpp>
+#include <opencv2/gapi.hpp>
+#include <opencv2/gapi/core.hpp>
+#include <opencv2/gapi/gpu/ggpukernel.hpp>
+#include <opencv2/gapi/streaming/onevpl/source.hpp>
+#include <opencv2/gapi/streaming/onevpl/data_provider_interface.hpp>
+#include <opencv2/gapi/streaming/onevpl/utils.hpp>
+#include <opencv2/highgui.hpp> // CommandLineParser
+#include <opencv2/gapi/ocl/core.hpp>
+
+const std::string about =
+    "This is an example presents decoding on GPU using VPL Source and passing it to OpenCL backend";
+const std::string keys =
+    "{ h help   |                           | Print this help message }"
+    "{ input    |                           | Path to the input video file. Use .avi extension }"
+    "{ output   | output.avi                | Path to the output RAW video file. Use .avi extension }";
+
+int main(int argc, char *argv[]) {
+    cv::CommandLineParser cmd(argc, argv, keys);
+    cmd.about(about);
+    if (cmd.has("help")) {
+        cmd.printMessage();
+        return 0;
+    }
+
+    // Get file name
+    const auto input = cmd.get<std::string>("input");
+    const auto output = cmd.get<std::string>("output");
+
+    // Make a OneVPL capture with default configuration
+    cv::Ptr<cv::gapi::wip::IStreamSource> cap = cv::gapi::wip::onevpl::make_default_onevpl_src(input);
+
+    // Build the graph
+    cv::GFrame in; // input frame from VPL source
+    auto bgr_gmat = cv::gapi::streaming::BGR(in); // conversion from VPL source frame to BGR UMat
+    auto out = cv::gapi::blur(bgr_gmat, cv::Size(4,4)); // ocl version of blur kernel
```

Comment:
please fix comment:
- ocl version is actually a kernel
- blur itself isn't a kernel, it is operation

Reply:
Done